### ISSUE TITLE
parsnp cannot find input sequences

### ISSUE THREAD

**Issue by ceya** [marbl/parsnp#34]
Hello,

I was trying to do a simple test run with the following code:
parsnp -p 2 -d test
and got the following: 
ERROR: no seqs provided, yet required. exit!

Does anyone know what went wrong?
Thanks!
Brooke

**Comment by gotero** [comment]
Were the sequences you wanted to align in the test directory?

**Comment by ceya** [comment]
Yes. They are multifasta files of DNA in the format
>strainA_1

>strainA_2
...

**Comment by gotero** [comment]
I believe you'll have to provide a reference genome with the -r option, -g option (for genbank file) or -r ! if you want parnsp to choose one of the sequences you provided at random to be used as a a reference for alignment

**Comment by ceya** [comment]
Yes! Thank you so much!

**Comment by ceya** [comment]
Sorry one more question, the output only gives me .ggr .tree and .xmfa
files, the vcf file is missing. Is this because I don't have a reference?

Here's the log:
Sequence 2 : test/103v_prokka.fasta
103v_prokka.fasta
Length:   2226274 bps
 GC:  839751.0
 AT: 1352597.0
Sequence 3 : test/103o_prokka.fasta
103o_prokka.fasta
Length:   2227173 bps
 GC:  832981.0
 AT: 1338232.0
Sequence 4 : test/103b_prokka.fasta
103b_prokka.fasta
Length:   2246911 bps
 GC:  848767.0
 AT: 1367800.0
d value:   300
q value:   30

Mum anchor size:   22.0
Number of MUM anchors found:   366
Number of MUMs found:   228
Total MUMs found((Anchors+MUMs)-filtered):   409

Random MUM length:   1
Minimum Cluster length:   21
Number of MUMs filtered:   185
Number of Clusters filtered:   183

Number of clusters created:   228
Average number of MUMs per cluster:   1
Average cluster length:   8846 bps
Cluster coverage in sequence 1:   92.0%
Cluster coverage in sequence 2:   92.0%
Cluster coverage in sequence 3:   92.9%
Cluster coverage in sequence 4:   91.0%
Total coverage among all sequences:   90.7%

 MUM anchor search elapsed time:   4.0s
 MUM coarsening elapsed time:   1.0s
 MUM filtering elapsed time:   0.0s
 MUM clustering elapsed time:   0.0s
 Inter-clustering elapsed time:   0.0s

**Comment by gotero** [comment]
You need to install Harvest Tools (https://harvest.readthedocs.io/en/latest/content/harvest-tools.html) and then you can run this command to create the vcf from the .ggr file: harvesttools -i parsnp.ggr -V parsnp.vcf

**Comment by barrantesisrael** [comment]
I'm having a related problem. When I downloaded the precompiled binary from the readthedocs, it showed no error, just the help, and it didn't read the inputs. 

Later, I installed using git, and the output was the following:

```
$ parsnp –g SF370.gbk –d targets -p 2 -c
0
 Cannot open reference file ! 
```

The genbank and fasta inputs are all ok.

**Comment by bkille** [comment]
This has been fixed in the current parsnp 1.5 version
